gc/CMakeLists.txt

```diff
@@ -167,6 +167,9 @@ add_library(omrgc STATIC
 	stats/ScavengerStats.cpp # TODO only compile if scavenger or VLHGC. Is this actually used by VLHGC?
 	stats/SweepStats.cpp
 
+	stats/ConcurrentGCStats.cpp
```

Comment:
Shouldn't these two files only be included if the proper define is set?

Reply:
I think this was an old compilation fix that's not needed any more. I'm not sure since I've squashed my history. I'll remove it.